PERFTEST.PS1

```diff
@@ -8,11 +8,11 @@ Param(
     
     [ValidateSet("Default", "Azure", "Detail", "Max")]
     [parameter(Mandatory=$false)] [string] $Config = "Default",
-    [parameter(Mandatory=$false)]  [Array] $ToolList = @("ntttcp", "latte", "cps", "ctstraffic")
+    [parameter(Mandatory=$false)]  [Array] $ToolList = @("ntttcp", "latte", "l4ping", "cps", "ctstraffic")
```

Comment:
should there be a validation check for the list?

Reply:
I think that's what the normalize_toollist function does. Is that what you mean?

Reply:
Leaving as is based on offline discussion.